pubspec.yaml

```diff
@@ -21,6 +21,7 @@ dependencies:
   mustache4dart: ^2.1.0
   package_config: '>=0.1.5 <2.0.0'
   path: ^1.3.0
+  quiver: ^0.27.0
```

Comment:
ah, we'd had an undeclared dep on quiver?

Reply:
Indeed.  I think we survived because tuple included it, but for some reason that has shifted to no longer work well with the SDK.